cmake/GammaRayMacrosInternal.cmake

```diff
@@ -106,6 +108,13 @@ macro(gammaray_embed_info_plist _target _plist)
   endif()
 endmacro()
 
+macro(gammaray_add_win_icon _sources)
+    if (WIN32)
+        configure_file(${_gammaray_macros_internal_location}/gammaray.rc.cmake gammaray.rc)
+        LIST(APPEND ${_sources} ${CMAKE_CURRENT_BINARY_DIR}/gammaray.rc)
```

Comment:
I guess as it is a macro I don't need to put it in the parent scope.
Or something like that
.